tests/results/test_floating_field_failing.html

```diff
@@ -1,6 +1,6 @@
 <form  method="post" >
     <div id="div_id_text_area" class="form-floating mb-3">
-        <textarea name="text_area" cols="40" rows="10" placeholder="text_area" class="textarea form-control is-invalid" required id="id_text_area" aria-invalid="true">
+        <textarea aria-describedby="id_text_area_error" name="text_area" cols="40" rows="10" placeholder="text_area" class="textarea form-control is-invalid" required id="id_text_area" aria-invalid="true">
```

Comment:
These ones adding references to errors I'm happy with. That was a new feature in 5.2.

Reply:
https://github.com/django-crispy-forms/crispy-bootstrap5/blob/main/crispy_bootstrap5/templates/bootstrap5/layout/field_errors.html This (and the block version) will need updating so that the new `aria-describedby` has a target to reference. 

See error when testing with the 'wave' accessibility checker:

![image](https://github.com/user-attachments/assets/95b76900-535b-4c8f-9893-3b3ed6e950f4)